scripts/update-tezos.sh

```diff
@@ -39,12 +37,10 @@ if [[ "$latest_upstream_tag" != "$our_tezos_tag" ]]; then
     git switch -c "$branch_name"
     echo "Updating Tezos to $latest_upstream_tag"
 
-    cd nix
-    niv update tezos -a ref="refs/tags/$latest_upstream_tag" -a rev="$latest_upstream_tag_hash"
-    niv update opam-repository -a rev="$opam_repository_tag" -a ref="$opam_repository_branch" -b "$opam_repository_branch"
+    python3 ./scripts/update-input.py tezos "$latest_upstream_tag_hash"
+    python3 ./scripts/update-input.py opam-repository "$opam_repository_tag"
```

Comment:
This one didn't work for me for some reason when I tried to simulate an update from `v12.3` to `v13.0` locally. Though `tezos` in flake.nix was updated correctly. Also, seems like `flake.lock` still requires an update. I guess the best way to do this is to run `nix flake lock --update-input tezos` and `nix flake lock --update-input opam-repository` afterwards